<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VehicleDialog</class>
 <widget class="QDialog" name="VehicleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vehicle</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="numberLabel">
     <property name="text">
      <string>Number</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="numberEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="compClassLabel">
     <property name="text">
      <string>Class</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="compClassEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="weightLabel">
     <property name="text">
      <string>Weight</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="weightEdit"/>
   </item>
   <item row="4" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QFrame" name="settingsFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QGroupBox" name="predictionDefaults">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Prediction defaults</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="2" column="0">
          <widget class="QLabel" name="weightAdjustmentLabel">
           <property name="text">
            <string>ET correction per 1lb of weight</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QDoubleSpinBox" name="weightAdjustmentSpinBox">
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="maximum">
            <double>0.999000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.001000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QDoubleSpinBox" name="windAdjustmentSpinBox">
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="minimum">
            <double>0.000000000000000</double>
           </property>
           <property name="maximum">
            <double>0.999000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.001000000000000</double>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="ProviderBox" name="textProviderComboBox"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="windAdjustmentLabel">
           <property name="text">
            <string>ET correction per 1MPH of wind</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="textProviderLabel">
           <property name="text">
            <string>Text Provider</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="textNumberLabel">
           <property name="text">
            <string>Text Number</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLineEdit" name="textNumberEdit"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QFrame" name="maintenanceFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QGroupBox" name="maintenanceInfo">
        <property name="title">
         <string>Maintenance Info</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="1" column="0">
          <widget class="QDateTimeEdit" name="lastOilChangeEdit"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="lastOilChangeLabel">
           <property name="text">
            <string>Last oil change</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QDateTimeEdit" name="lastTireChangeEdit"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lastOilRunsLabel">
           <property name="text">
            <string>Runs 0</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="lastTireChangeLabel">
           <property name="text">
            <string>Last tire change</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="lastTireRunsLabel">
           <property name="text">
            <string>Runs 0</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ProviderBox</class>
   <extends>QComboBox</extends>
   <header>providerbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>numberEdit</tabstop>
  <tabstop>compClassEdit</tabstop>
  <tabstop>weightEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>VehicleDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>VehicleDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
